### PR TITLE
Remaining implementation of the NativeTheme API

### PR DESCRIPTION
**Pull request by konstantingross**
'shouldUseHighContrastColors', 'shouldUseInvertedColorScheme', get/set 'themeSource' and 'updated' event added

### CODE REVIEW

File: ElectronNET.API/NativeTheme.cs [hunk 1]
```diff
@@ -32,11 +33,114 @@ internal static NativeTheme Instance
...
 
         /// <summary>
-        /// A `Boolean` for if the OS / Chromium currently has a dark mode enabled or is
-        /// being instructed to show a dark-style UI.If you want to modify this value you
-        /// should use `themeSource` below.
+        /// Checks if the new ThemeSource is valid.
+        /// </summary>
+        /// <param name="themeSource">The new ThemeSource to check.</param>
+        /// <returns>True, if is a valid ThemeSource.</returns>
+        internal bool IsValidThemeSource(string themeSource)
```
Comment (GregorBiswanger) [inline ElectronNET.API/NativeTheme.cs]: Can you offer this to an enum? Is type-safe and guides the developer in the flow ...

File: ElectronNET.API/NativeTheme.cs [hunk 2]
```diff
@@ -32,11 +33,114 @@ internal static NativeTheme Instance
...
+        /// <description>Dark Mode: SetThemeSource("dark");</description>
+        /// </item>
+        /// <item>
+        /// <description>Light Mode: SetThemeSource("light");</description>
+        /// </item>
+        /// </list>
+        /// Your application should then always use <see cref="ShouldUseDarkColorsAsync"/> to determine what CSS to apply.
+        /// </summary>
+        /// <param name="themeSource">The new ThemeSource.</param>
+        public void SetThemeSource(string themeSource)
```
Comment (GregorBiswanger) [inline ElectronNET.API/NativeTheme.cs]: Here the same please...

File: ElectronNET.API/NativeTheme.cs [hunk 3]
```diff
@@ -32,11 +33,114 @@ internal static NativeTheme Instance
...
+        {
+            // Check for supported themeSource, otherwise it sets the default
+            if (!IsValidThemeSource(themeSource))
```
Comment (GregorBiswanger) [inline ElectronNET.API/NativeTheme.cs]: No must.. but it is nicer to check for a truth than for an if not equal then... (clean code)

Comment (konstantingross) [reply ElectronNET.API/NativeTheme.cs]: I didn't know if I was allowed to turn everything completely upside down :D With an enum I can delete this method.

**Pull request merged**